src/projectscene/view/clipsview/clipslistmodel.cpp

```diff
@@ -188,10 +204,11 @@ void ClipsListModel::update()
 
     updateItemsMetrics();
 
-    //! NOTE We need to update the selected item
-    //! to take a pointer to the item from the new list
-    m_selectedItem = nullptr;
-    onSelectedClip(selectionController()->selectedClip());
+    //! NOTE We need to update the selected items
+    //! to take pointers to the items from the new list
+    m_selectedItems = QList<ClipListItem*>();
```

Comment:
better to call `m_selectedItems.clear()`

Reply:
fixed